macros/plotTimeSeries.py

```diff
@@ -1,22 +1,17 @@
 #!/usr/bin/env python
 
-def makeInputList(chamberName, scanType):
-    ofname = getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt'
-    with open(template,"rt") as fin:
-        with open(ofname, "wt") as fout:
-            for line in fin:
-                fout.write(line.replace("GEMINI", chamberName)) 
-
-def makePlots(chamberName, scanType, vt1bump, elog_path):
-    call_command = 'gemPlotter.py --infilename='+getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=threshold --make2D --alphaLabels -c -a'
+def makePlots(chamberName, anaType, vt1bump, elog_path):
+    call_command = 'gemPlotter.py --skipBadFiles --infilename='+getDirByAnaType(anaType,chamberName)+'listOfScanDates.txt --anaType=scurveAna --branchName=threshold --make2D --alphaLabels -a --axisMax=10'
+    os.system(call_command)
```

Comment:
please consider refactoring the gemPlotter so it can be called in a pythonic way

Reply:
For sure that will not come in this PR or an upcoming PR on a short-time scale.  The bulk of the work being done by `gemPlotter.py` is already "pythonic" since it is separated into two functions:

- `arbitraryPlotter(...)`,
- `arbitraryPlotter2D(...)`.

If one wanted they could import these, and then build around importing that; if this is desired then these functions could be moved to `anautilities.py` or an independent `plottingUitilties.py` file.

But additional re-factoring of this tool I think is a lower priority compared to other critical tasks at present.